buildSrc/build.gradle.kts

```diff
@@ -24,8 +24,20 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath("com.github.jk1:gradle-license-report:1.16")
+    }
+}
+
 plugins {
+    java
     `kotlin-dsl`
+    id("com.github.jk1.dependency-license-report") version "1.16"
```

Comment:
Why do we need the `license-report` plugin specified in both `buildscript` and `plugins`?

Also, I believe that `version` is redundant if we specify the plugin in `buildscript`.

Reply:
We apply the plugin as a part of a build, and we need its code to build a custom report renderer.